platforms/platform-googlebusiness/README.md

```diff
@@ -0,0 +1,143 @@
+---
+title: 'Google Business Messages Platform Integration'
+excerpt: 'The Google Business Messages platform integration allows you to build custom Google Business Bots using Jovo.'
+---
+
+# Google Business Messages Platform Integration
+
+The Google Business Messages [platform integration](https://v4.jovo.tech/docs/platforms) allows you to build custom Google Business bots using Jovo.
+
+
+## Getting Started
+
+You can install the plugin like this:
+
+```sh
+$ npm install @jovotech/platform-googlebusiness
+```
+
+Add it as plugin to your [app configuration](https://v4.jovo.tech/docs/app-config), e.g. `app.ts`:
+
+```typescript
+import { App } from '@jovotech/framework';
+import { GoogleBusinessPlatform } from '@jovotech/platform-googlebusiness';
+// ...
+
+const app = new App({
+  plugins: [
+    new GoogleBusinessPlatform(),
+    // ...
+  ],
+});
+```
+
+## Configuration
+
+You can configure the Google Business platform in the [app configuration](https://v4.jovo.tech/docs/app-config), for example `app.ts`:
+
+```typescript
+import { GoogleBusinessPlatform } from '@jovotech/platform-googlebusiness';
+
+// ...
+
+const app = new App({
+  plugins: [
+    // For example NLU integration
+  ],
+  sessionExpiresAfterSeconds: 900
+});
+```
+
+Options include:
+
+- `plugins`: Add an [NLU integration](#nlu-integration) here.
+- `sessionExpiresAfterSeconds`: Defines when a request should be treated as a new session. Take a look at [session data](#session-data) for more information.
```

Comment:
Same situation as the `session`-object in the configuration of `FacebookMessengerPlatform`.